docs/source/en/model_doc/mobilenet_v2.md

```diff
@@ -14,61 +14,93 @@ rendered properly in your Markdown viewer.
+with torch.no_grad():
+  logits = model(**inputs).logits
+predicted_class_id = logits.argmax(dim=-1).item()
 
-## Resources
+class_labels = model.config.id2label
+predicted_class_label = class_labels[predicted_class_id]
+print(f"The predicted class label is: {predicted_class_label}")
+```
 
-A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with MobileNetV2.
+</hfoption>
+</hfoptions>
 
-<PipelineTag pipeline="image-classification"/>
+<!-- Quantization - Not applicable -->
+<!-- Attention Visualization - Not applicable for this model type -->
 
-- [`MobileNetV2ForImageClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/image-classification) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/image_classification.ipynb).
-- See also: [Image classification task guide](../tasks/image_classification)
+## Notes
 
-**Semantic segmentation**
-- [Semantic segmentation task guide](../tasks/semantic_segmentation)
+-   **Checkpoint Naming:** Classification checkpoints often follow `mobilenet_v2_{depth_multiplier}_{resolution}`, like `mobilenet_v2_1.4_224`. Segmentation checkpoints (using DeepLabV3+ head) might have names like `deeplabv3_mobilenet_v2_{depth_multiplier}_{resolution}`.
+-   **Variable Input Size:** Like V1, the model works with images of different sizes (minimum 32x32), handled by [`MobileNetV2ImageProcessor`].
+-   **1001 Classes (Classification):** ImageNet-1k pretrained classification models output 1001 classes (index 0 is background).
+-   The segmentation models use a [DeepLabV3+](https://huggingface.co/papers/1802.02611) head which is often pretrained on datasets like [PASCAL VOC](https://huggingface.co/datasets/merve/pascal-voc).
+-   **Padding Differences:** Similar to V1, original TensorFlow checkpoints had dynamic padding. The HF PyTorch implementation uses static padding by default. Enable dynamic padding (TF behavior) via `tf_padding=True` in [`MobileNetV2Config`].
+    ```python
+    from transformers import MobileNetV2Config
 
-If you're interested in submitting a resource to be included here, please feel free to open a Pull Request and we'll review it! The resource should ideally demonstrate something new instead of duplicating an existing resource.
+    # Example: Load config with dynamic padding enabled
+    config = MobileNetV2Config.from_pretrained("google/mobilenet_v2_1.4_224", tf_padding=True)
+    ```
+-   **Unsupported Features:**
+    -   The HF implementation uses global average pooling, not the optional fixed 7x7 average pooling from the original paper.
+    -   Extracting specific intermediate hidden states (e.g., from expansion layers 10/13) requires `output_hidden_states=True` (returning all states).
+    -   For segmentation models, the final convolution layer of the backbone is computed even though the DeepLabV3+ head doesn't use it.
```

Comment:
```suggestion
-   The Transformers implementation does not support the following features.
    -   Uses global average pooling instead of the optional 7x7 average pooling with stride 2. For larger inputs, this gives a pooled output that is larger than a 1x1 pixel.
    -   `output_hidden_states=True` returns *all* intermediate hidden states. It is not possible to extract the output from specific layers for other downstream purposes.
    - Does not include the quantized models from the original checkpoints because they include "FakeQuantization" operations to unquantize the weights.
    -   For segmentation models, the final convolution layer of the backbone is computed even though the DeepLabV3+ head doesn't use it.
```